### DISCUSSION TITLE
Update self-hosted runners installing documentation

### DISCUSSION THREAD

**Discussion by maduq1** [community/community#12601] [Actions] [answered]
Hi! We are using actions in self-hosted runners and we encountered the following issue:
https://github.com/google-github-actions/auth/issues/142

We thought it was some sort of problem with that specific action but then we discovered the problem was that our runners were not correctly updated. We've seen that in our machine, the runner updated itself a couple of times, but got stuck in v2.283, while latest version is 2.288. 

In the documentation, when trying to add a new runner, it provides some instructions, and they also advice to download version 2.283 without any other kind of updating or version notification.

This has become a big problem because the runners have been updated to support only node16 and we are receiving compatibility errors related to node. 

So TLDR:
- Update self-hosted runners documentation so the latest version is installed
- Why the autoupdate of the self-hosted runners got stuck in 2.283 without any further notice?

Thanks a lot in advance, my team and I ❤️  github actions!!

**Comment by ethomson** [comment, marked as answer]
Hi @maduq1 - thanks for letting us know.

> Why the autoupdate of the self-hosted runners got stuck in 2.283 without any further notice?

There was a problem in v2.283 where it was not able to perform the automatic update in a very narrow set of circumstances.

We emailed the owner of every organization or enterprise that had runners stuck and unable to update with instructions about how to proceed. I apologize that the email you didn't get the email, but I'm glad that you found the problem yourself and resolved it. (And do let me know if you don't think that your organization/enterprise contact received the email and we can investigate so that we get this right next time.)
